near-plugins/src/test_utils.rs

```diff
@@ -15,8 +17,8 @@ pub(crate) fn get_context() -> VMContext {
         account_locked_balance: 0,
         storage_usage: 10_000,
         attached_deposit: 0,
-        prepaid_gas: 10u64.pow(18),
-        random_seed: vec![0, 1, 2],
+        prepaid_gas: Gas(10u64.pow(18)),
```

Comment:
It's better to already use a new available constant. However, it seems we are using a super big value for that:)

```suggestion
        prepaid_gas: 1_000_000 * Gas::ONE_TERRA,
```

Reply:
Done in 114f74d.